test/cli.test.js

```diff
@@ -109,6 +109,33 @@ describe("CLI server command", function() {
    });
 });
 
+describe("CLI build timeout", function() {
+   var exec = execInDir(testRepoDir);
+
+   it("sets the build timeout", function(done) {
+      var cimpler = new Cimpler({
+         plugins: {
+            cli: {
+            }
+         },
+         httpPort: httpPort,
+         testMode: true  // Don't console.log() anything
+      });
+      var timeout = 1 * 1000; // 1 second timeout
+      cimpler.on('buildFinished', function(build) {
```

Comment:
Just for my understanding, this could have been `'buildAdded'` or `'buildStarted'`?

Reply:
I think you're right. This proves the argument makes it through... but not that timeout is respected,

Reply:
Yeah, I was having a hard time testing that the timeout is respected. I would be happy to collab and maybe get a test written for that if either of you wanted.

Reply:
Idk how important that test would be.

Reply:
I think it's important to test that both the setting is set and that the setting is used. I added a test!